securitycenter/snippets/snippets_mute_config.py

```diff
@@ -234,6 +234,35 @@ def set_unmute_finding(finding_path: str) -> None:
 # [END securitycenter_set_unmute]
 
 
+# [START securitycenter_set_mute_undefined]
+def set_undefined_finding(finding_path: str) -> None:
+    """
+      Reset mute state of an individual finding.
+      Resetting a finding that isn't muted or unmuted has no effect.
+      Various mute states are: UNDEFINED/MUTE/UNMUTE.
+    Args:
+        finding_path: The relative resource name of the finding. See:
+        https://cloud.google.com/apis/design/resource_names#relative_resource_name
+        Use any one of the following formats:
+        - organizations/{organization_id}/sources/{source_id}/finding/{finding_id},
+        - folders/{folder_id}/sources/{source_id}/finding/{finding_id},
+        - projects/{project_id}/sources/{source_id}/finding/{finding_id}.
+    """
+    from google.cloud import securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+
+    request = securitycenter.SetMuteRequest()
```

Comment:
Does this API support to be set up declaratively instead of imperatively?

Something like this, instead of mutating the request object via property assignments.

```py
request = securitycenter.SetMuteRequest(
  name=finding_path,
  mute=securitycenter.Finding.Mute.UNDEFINED,
)
```

Reply:
I think both are okay, but I'd prefer to follow above samples of `set_unmute_finding` and `set_mute_finding` to keep style consistency.

Reply:
In the [style guide](https://googlecloudplatform.github.io/samples-style-guide/#pattern),  the examples arrange requests in this way. I think it's easier to read and reason about, but I'll let other reviewers share their opinion around this.

For consistency, I would update other samples to match, either in this PR or a follow up.